core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -435,10 +436,22 @@ void print_stack_arm32(int level, struct unwind_state_arm32 *state,
 		       vaddr_t exidx, size_t exidx_sz,
 		       vaddr_t stack, size_t stack_size)
 {
+#ifdef CFG_CORE_SYMS
```

Comment:
Minor: moving the #ifdef outside the function may be more legible, and same for arm64 below.

Reply:
just to clarify: you are proposing to have two variants of `print_stack_arm32()`: with and without symbols support.  Am I correct?

Reply:
Yes.